gradle/libs.versions.toml

```diff
@@ -30,7 +30,7 @@ sl4j = "2.0.7"
 junit = "4.13.2"
 kotestAsserions = "4.6.3"
 jsoup = "1.14.3"
-arrow = "11.0.0"
+arrow = "14.0.2"
```

Comment:
I'm sorry for the late response. We want to merge NullVector reading support for version 0.13.0-RC1 soon, but  in the meantime arrow was updated to 15.0.0. (https://github.com/Kotlin/dataframe/pull/590)
Could you update it here too?

Reply:
Done